src/Helpers.purs

```diff
@@ -16,3 +19,13 @@ parseJsonStringifyNumbers s = do
   _ <- parseJson s
   -- valid json ensured at this point
   parseJson $ jsonTurnNumbersToStrings s
+
+explain :: forall a e. e -> Maybe a -> Either e a
```

Comment:
Kind signature for polymorphic parameters if possible. Also, isn't this the same as `note`? https://pursuit.purescript.org/packages/purescript-either/5.0.0/docs/Data.Either#v:note